tests/integrations/aiohttp/test_aiohttp.py

```diff
@@ -55,7 +55,7 @@ async def hello(request):
     assert request["url"] == "http://{host}/".format(host=host)
     assert request["headers"] == {
         "Accept": "*/*",
-        "Accept-Encoding": "gzip, deflate",
+        "Accept-Encoding": mock.ANY,
```

Comment:
This is because `brotli` is not available for Python versions such as 3.7 etc and the output changes based on the availability of `brotli` module (if it is there a `, br` is added to the end of the list.